210-on-disk-representation.md

```diff
@@ -0,0 +1,11 @@
+---
+title: On Disk Representation of a CNAB bundle
+weight: 210
+---
+
+# On Disk Representation of a CNAB bundle
+
+TODO:
```

Comment:
Since this is still undefined, should this file be removed from this PR and filled in during a follow-up PR?